cross/arm32_ci_script.sh

```diff
@@ -215,7 +219,7 @@ function cross_build_corefx_with_docker {
     else
         exit_with_error "ERROR: unknown buildArch $__buildArch" false
     fi
-    __dockerCmd="sudo docker run --privileged -i --rm -v $__currentWorkingDirectory:/opt/corefx -w /opt/corefx $__dockerImage"
+    __dockerCmd="sudo docker run ${__dockerEnvironmentVariables} --privileged -i --rm -v $__currentWorkingDirectory:/opt/corefx -w /opt/corefx $__dockerImage"
 
     if [ $__skipRootFS == 0 ]; then
```

Comment:
We still generate rootfs for Tizen armel and I would like to maintain this to run CI with the latest rootfs later.